lib/aruba/api/core.rb

```diff
@@ -25,8 +25,8 @@ def aruba
       # This will only clean up aruba's working directory to remove all
       # artifacts of your tests. This does NOT clean up the current working
       # directory.
-      def setup_aruba
-        Aruba::Setup.new(aruba).call
+      def setup_aruba(clobber: true)
+        Aruba::Setup.new(aruba).call(clobber: clobber)
```

Comment:
This syntax is the incompatible one.

Reply:
Yes, but it should be fine on master since we dropped support for 1.8.

Reply:
Ah bummer, I forgot this won't work with 1.9 either 😞